packages/@glimmer/debug/lib/debug.ts

```diff
@@ -183,7 +186,8 @@ export type OperandOptionsA<O extends AnyOperand> = O extends [
   options: infer Options,
 ]
   ? Options
-  : {};
+  : // eslint-disable-next-line @typescript-eslint/no-empty-object-type
```

Comment:
I didn't always try to figure out what `{}` could be replaced with without creating more type
errors, but it's a good thing to follow-up with.